### PR TITLE
added length.md documentation

### LINKED ISSUE #4371: [Term Entry] Dart Maps .length()
### Reason/inspiration (optional)

"We would like a new term entry under Dart [Maps](https://www.codecademy.com/resources/docs/dart/concept/map) for the Dart Map `.length()` method. 
Provide the entry in a markdown file under `docs/content/dart/concept/map/terms/length/length.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - A syntax section with a pseudo code block with an explanation of it.
    - An example showing the method in use.

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by aarosan**
### Description

In this pull request, I added directories and a length.md file as shown below:
content/dart/concepts/map/terms/length/length.md

I followed the term-entry-template to add information about the length property in dart.

### Issue Solved

The issue is complete to my knowledge. If there is anything you would like changed, please let me know and I am happy to fix it!

Closes #4371 

### Type of Change

- Adding a new entry
- Updating the documentation (the only file that was edited was the file that was created: length.md)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/map/terms/length/length.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+In the following example, the `.length` property is used to get the key-value pairs of a Map instance that uses similar DataTypes.
+
+```dart
+Map<String, String> countries = {
+    'USA': 'United States',
+    'IRE': 'Ireland',
+    'ITLY': 'Italy'
+};
+print(countries.length);
+```
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:38]: The entire code block should be executable as it is. You need to wrap this code inside the main function.

File: content/dart/concepts/map/terms/length/length.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Example 2
+
+The `.length` property can take different DataTypes for its key-value pairs as well.
+
+```dart
+Map<String, int> fruits = {
+    'Apple': 3,
+    'Banana': 2,
+    'Mango': 9
+}
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:54]: The print statement is not there so there would be no output. Please add the print statement so it'll be a valid code and output block.

File: content/dart/concepts/map/terms/length/length.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+## Example 3
+
+The `.length` property will take in account any duplicate keys and will not include them.
+
+```dart
+Map<String, int> fruits = {
+    'Apple': 3,
+    'Banana': 2,
+    'Apple': 1
+}
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:71]: Same goes for this code block.

File: content/dart/concepts/map/terms/length/length.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+Title: 'length'
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:2]:
```suggestion
Title: '.length()'
```

File: content/dart/concepts/map/terms/length/length.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+Tags:
+  - 'Dart'
+  - 'Properties'
+  - 'Map'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, the **`.length`** property is a utility for determining the number of key-value pairs contained within a Map instance. It enables developers to efficiently manage and manipulate data structures by providing a quick and reliable means of accessing the size of a Map.
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:16]:
```suggestion
In Dart, the **`.length()`** property is a utility for determining the number of key-value pairs contained within a Map instance. It enables developers to efficiently manage and manipulate data structures by providing a quick and reliable means of accessing the size of a Map.
```

File: content/dart/concepts/map/terms/length/length.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+## Syntax
+
+```pseudo
+print(<Map-variable>.length);
+```
+
+- `print();`: A built-in function that outputs data to the console.
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:24]:
```suggestion
```
No need for this description.

File: content/dart/concepts/map/terms/length/length.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, the **`.length`** property is a utility for determining the number of key-value pairs contained within a Map instance. It enables developers to efficiently manage and manipulate data structures by providing a quick and reliable means of accessing the size of a Map.
+
+## Syntax
+
+```pseudo
+print(<Map-variable>.length);
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:21]:
```suggestion
Map-variable.length
```

File: content/dart/concepts/map/terms/length/length.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+- `print();`: A built-in function that outputs data to the console.
+- `<Map-variable>`: Represents a variable that holds a Map object.
```
Comment (avdhoottt) [inline content/dart/concepts/map/terms/length/length.md:25]:
```suggestion
- `Map-variable`: Represents a variable that holds a Map object.
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Nice entry @aaron-ross-sanchez. I have suggested some changes, please make them.

**Comment by aaron-ross-sanchez** [comment]
I made all of the suggested changes, please let me know what else needs to be adjusted! Thank you!

**Review by avdhoottt — APPROVED** [review]
Look good for the second review!!

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Pull request merged**